src/HomeWork_8_2/BaseConverter.java

```diff
@@ -0,0 +1,40 @@
+//Напишите класс BaseConverter для конвертации градусов по Цельсию, Кельвину ,  и Фаренгейту между собой.
+// У класса должен быть 1 публичный метод convert, который и делает конвертацию.
+// Вводить данные и выводить результат операций с указанием шкалы градусов (F / K / C на конце). Использовать enum
+//Соотношения температур:
+//• <Температура по Цельсию>
+//• <Температура по Цельсию> + 273.15 —-> по Кельвину
+//• 1.8 * <Температура по Цельсию> + 32 —-> по Фаренгейту
+
+package HomeWork_8_2;
+
+public class BaseConverter {
+
+    public void convert(double temperature, Scale inputScale) {
```

Comment:
Sense was in converting all temperatures between each other. C -> K,F; K -> C, F; F -> C,K